SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -586,38 +611,32 @@ void ASpatialFunctionalTest::SetupClientPlayerRegistrationFlow()
 	));
 }
 
-void ASpatialFunctionalTest::CrossServerRegisterAutoDestroyActor_Implementation(AActor* ActorToAutoDestroy)
-{
-	RegisterAutoDestroyActor(ActorToAutoDestroy);
-}
-
-void ASpatialFunctionalTest::ServerRegisterAutoDestroyActor_Implementation(AActor* ActorToAutoDestroy)
```

Comment:
These functions need to be there to allow any workers to mark an actor for auto destroy

Reply:
Discussed this with Nuno and decided the use case should be to create and then register actor for auto destroy so should not need cross-server calls. Added an error log in case this is used incorrectly.